genawaiter-proc-macro/src/visit.rs

```diff
@@ -15,7 +15,9 @@ impl VisitMut for YieldReplace {
                 let tkns: TokenStream2 = syn::parse2(m.mac.tokens.clone())
                     .expect("parse of TokensStream failed");
 
-                let co_call = quote! { yield_!(@emit => __private_co_arg__, #tkns) };
+                let co_call = quote! {
+                    ::genawaiter::yield_!(@emit => __private_co_arg__, #tkns)
```

Comment:
this allows user to only import the macro for creating generators. I'm not sure how I feel about that it's nice and ergonomic and as long as its documented well its discoverable but it feels sneaky??

Reply:
I agree, I think it's a bit "sneaky" as well, and it would probably cause some minor papercuts like IDEs not autocompleting the `yield_` macro. I would probably revert that part.